torch/csrc/jit/frontend/lexer.h

```diff
@@ -1,13 +1,16 @@
 #pragma once
 #include <c10/macros/Macros.h>
 #include <c10/util/Exception.h>
+#include <c10/util/irange.h>
 #include <torch/csrc/Export.h>
 #include <torch/csrc/jit/frontend/parser_constants.h>
 #include <torch/csrc/jit/frontend/source_range.h>
 #include <torch/csrc/jit/frontend/strtod.h>
 #include <algorithm>
+#include <array>
```

Comment:
```suggestion
#include <array>
#include <cctype>
```
For the std::ischar and std::isalpha